pgrx-tests/tests/ui/elog/semicolon-terminators/ereport_detail.stderr

```diff
@@ -0,0 +1,12 @@
+error: The alternative invocation style requires component sections to be terminated with a semicolon.
+  --> tests/ui/elog/semicolon-terminators/ereport_detail.rs:7:5
+   |
+7  | /     ereport!(
+8  | |         loglevel = PgLogLevel::LOG;
+9  | |         errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
+10 | |         message = "...";
+11 | |         detail = "detail",
+12 | |     );
```

Comment:
The semicolon-oriented style is a fairly novel style, syntactically, and if this is what is necessitated by this approach, I think it is useful to step back and ask why we are requiring it to look like that?